opensquirrel/parser/libqasm/parser.py

```diff
@@ -172,20 +228,23 @@ def circuit_from_string(self, s: str) -> Circuit:
         # Parse statements
         ir = IR()
         for statement in ast.block.statements:
-            if "measure" in statement.name:
-                generator_f_measure = self.get_measure_f(statement.name)
-                expanded_args = Parser._get_expanded_measure_args(statement.operands, register_manager)
-                for arg_set in expanded_args:
-                    ir.add_measure(generator_f_measure(*arg_set))
-            elif "reset" in statement.name:
-                generator_f_reset = self.get_reset_f(statement.name)
-                expanded_args = Parser._get_expanded_reset_args(statement.operands, register_manager)
-                for arg_set in expanded_args:
-                    ir.add_reset(generator_f_reset(*arg_set))
+            instruction_generator: Callable[..., Statement]
```

Comment:
This does not need to be declared (it doesn't hurt either, I guess)

Reply:
Removed.